docs/examples/tutorial/cython_tutorial/setup_primes_pure.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup
+from Cython.Build import cythonize
+
+setup(
+    ext_modules=cythonize(
+        ['primes_pure.py',              # Cython code file with primes() function
```

Comment:
You called it primes.py elsewhere

Reply:
Oh yeah, my bad

Reply:
[here](https://github.com/cython/cython/pull/4226/commits/5dc05c7ca83f90570825c589d5049ace3ec61ac0)